src/ConfigMenu.cc

```diff
@@ -187,7 +187,7 @@ void setupFocusMenu(FbTk::Menu& menu, ConfigMenu::SetupHelper& sh, _Cmd& save_rc
             "Keep Head", "Only revert focus on same head"),
             sh.rm.getResource<bool>(sh.screen.name() + ".focusSameHead"),
             save_rc));
-    } catch (FbTk::ResourceException e) {
+    } catch (FbTk::ResourceException &e) {
```

Comment:
is this for "performance" reasons?

Reply:
Don't know; didn't come up this time around either in #76 or in the revised version of this PR.